rest/messaging/v1/a2p_brand_registrations.go

```diff
@@ -51,7 +51,7 @@ func (c *ApiService) CreateBrandRegistrations(params *CreateBrandRegistrationsPa
 	headers := make(map[string]interface{})
 
 	if params != nil && params.A2pProfileBundleSid != nil {
-		data.Set("A2pProfileBundleSid", *params.A2pProfileBundleSid)
+		data.Set("A2PProfileBundleSid", *params.A2pProfileBundleSid)
```

Comment:
On L27 this needs to fixed as well.
`A2pProfileBundleSid *string `json:"A2pProfileBundleSid,omitempty"``